eng/common/scripts/Package-Properties.ps1

```diff
@@ -320,7 +320,9 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
     # packages. We should never return NO validation.
     if ($packagesWithChanges.Count -eq 0) {
         $packagesWithChanges += ($allPackageProperties | Where-Object { $_.ServiceDirectory -eq "template" })
-        $packagesWithChanges[0].IncludedForValidation = $true
+        foreach ($package in $packagesWithChanges) {
```

Comment:
While I'm not opposed to this, I'm curious if you think there is enough value to do all over just one?

Reply:
Are per our conversation this adds all template libraries and, for Java, there's 3 and they all need to be set.